### PR TITLE
Api refactor

### PR DESCRIPTION
**Pull request by dianacamacho**
Quite a few updates since the first version. Updated controllers/routes with versioned api for the translations, some schema updates. 

Deploying this will be a breaking change for fotofluent. The endpoint for translations will need to be updated to the new api route, example url: `/api/v1/translations?language_code=it-IT&grouping=colors&limit=1`
- `language_code`, `grouping`, and `limit` params are optional, default is german (de-DE), all groupings, and limit of 1

### CODE REVIEW

File: app/helpers/api/v1/translations_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Api::V1::TranslationsHelper
```
Comment (rkillackey) [inline app/helpers/api/v1/translations_helper.rb]: Can remove

File: app/controllers/words_controller.rb [hunk 2]
```diff
@@ -25,43 +19,54 @@ def edit
...
+  def word_service
+    @word_service ||= WordService.new(word_service_params)
+  end
+
+  def word_service_params
+    @word_service_params ||= WordServiceDecanter.decant(params)
+  end
+
+  def word_params
+    params.require(:word).permit(
```
Comment (rkillackey) [inline app/controllers/words_controller.rb]: Do we need to specify strong params if we're using decanter?

File: app/assets/javascripts/api/v1/translations.coffee [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (rkillackey) [inline app/assets/javascripts/api/v1/translations.coffee]: Can remove

File: app/assets/stylesheets/api/v1/translations.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the api/v1/translations controller here.
```
Comment (rkillackey) [inline app/assets/stylesheets/api/v1/translations.scss]: Can remove

**Review by rkillackey — APPROVED** [review]
This looks great! There are a few files that were auto-generated you can remove if you want. Do you want to update the endpoint on the client side?

**Comment by dianacamacho** [comment]
@rkillackey yup, i'll update the the endpoint. thanks!

**Comment by dianacamacho** [comment]
@rkillackey also made the client side extension update. let me know when you want to push that to the chrome store, so we can make sure to deploy this api change to heroku at the same time!

**Pull request merged**